app/assets/stylesheets/govuk_publishing_components/components/_accordion.scss

```diff
@@ -1,5 +1,13 @@
 @import "govuk/components/accordion/accordion";
 
+.gem-c-accordion {
+  .govuk-accordion__section-heading {
+    @include govuk-media-query($until: desktop) {
```

Comment:
The default at the moment seems to be `1.25` for both mobile and desktop. How did we come up with `1.6`?

Reply:
When it's at 1.6 the height of the touch area is in line with the accordion in the Design System.